deploy_pgcluster.yml

```diff
@@ -263,6 +264,9 @@
         (cluster_vip is defined and cluster_vip | length > 0)
 
     # optional
+    - role: postgresql-function
```

Comment:
There is no need to put this part of the code in a separate role, it is enough to do it as an additional task to the "pgbouncer" role.

Reply:
Agreed, I'll move that part to the config role